components/Buttons/src/MDCButton.m

```diff
@@ -632,7 +632,7 @@ - (CGFloat)borderWidthForState:(UIControlState)state {
   if (borderWidth != nil) {
     return (CGFloat)borderWidth.doubleValue;
   }
-  return 0;
+  return (CGFloat)[_borderWidths[@(UIControlStateNormal)] doubleValue];
```

Comment:
Please account for UIButton's behavior around `UIControlStateDisabled | UIControlStateHighlighted`

See https://github.com/material-components/material-components-ios/pull/5919 for more details.

Reply:
Done